helicone-node/async_logger/HeliconeLogger.ts

```diff
@@ -121,18 +118,18 @@ export class HeliconeLogBuilder {
 }
 
 export class HeliconeLogger {
-  protected heliconeConfiguration: IHeliconeConfiguration;
+  protected options: IHeliconeAsyncClientOptions;
   public helicone: Helicone;
   private logger: HeliconeAsyncLogger;
 
-  constructor(heliconeConfiguration: IHeliconeConfiguration) {
-    this.heliconeConfiguration = heliconeConfiguration;
-    this.helicone = new Helicone(heliconeConfiguration);
-    this.logger = new HeliconeAsyncLogger(heliconeConfiguration);
+  constructor(options: IHeliconeAsyncClientOptions) {
+    this.options = options;
+    this.helicone = new Helicone(options);
+    this.logger = new HeliconeAsyncLogger(options);
   }
 
-  submit(request: HeliconeLogBuilder): Promise<AxiosResponse<any, any>> {
+  async submit(request: HeliconeLogBuilder): Promise<Response> {
     const asyncLogRequest = request.build();
-    return this.logger.log(asyncLogRequest, Provider.CUSTOM_MODEL);
+    return await this.logger.log(asyncLogRequest, Provider.CUSTOM_MODEL);
```

Comment:
why does this need to await now?

Reply:
I think it should have always been awaited. The user can just decide to not await, but having it here gives them the option